api/src/test/resources/jwk-content1.json

```diff
@@ -0,0 +1,38 @@
+{
+  "iss": "https://server.example.com",
+  "sub": "24400320",
+  "unique_username": "jdoe@example.com",
+  "preferred_username": "jdoe",
+  "aud": "s6BhdRkqt3",
+  "exp": 1311281970,
+  "iat": 1311280970,
+  "auth_time": 1311280969,
+  "roles": [
+    "role-in-realm",
+    "user",
+    "manager"
+  ],
+  "groups": ["group1", "group2"],
+  "resource_access": {
```

Comment:
I'm expecting that the security integration layer is what is handling the mapping from the target microservice based on it's CDI name and declared role requirements.